### ISSUE TITLE
[MAS1.3.3] [Visual Requirements-Debug] Tool tips are not defined for “Previous, Next and Show diff” controls 

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1817]
**Actual Result:**
Tool tips are not defined for “Previous, Next and Show diff” controls present in the Debug mode 

**Expected Result:**
Tool tips should be defined for “Previous, Next and Show diff” controls

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Welcome tab screen gets opened
3. Bot screen upon connecting via azure/teams
4.Debug screen gets appeared on emulator 
5. Navigate to the “Previous, Next and Show diff” controls
6. Observe the issue

**User Impact:**
Users who rely on AT will be affected if the   Instructions provided for understanding and operating content do not rely solely on sensory characteristics of components such as shape, size, visual location, orientation, or sound

**Test Environment:**
OS Build: Version 1903(OS Build 18362.239)
Application: Bot Framework V4 Emulator

**MAS Reference Link:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={1823da5a-e8fb-4e63-b881-7eccb312c358}
[MAS1.3.3_Debug.zip](https://github.com/microsoft/BotFramework-Emulator/files/3569215/MAS1.3.3_Debug.zip)

**Comment by Kumar2608** [comment]
Verified in latest Bot emulator version -4.5.2-79718 . Issue got fixed hence we are closing this bug.
Tooltips are defined for "Previous ,Next and Show diff controls"
